src/platforms/react-native/manual-setup/native-init.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Native Initialization
+sidebar_order: 4
+description: "Learn how to manually initialize the native SDKs."
+---
+
+By default, the React Native SDK will initialize the native SDK underneath when the `init` method is called on the JS layer. This currently has the limitation of not capturing native crashes that occur before the `init` method is called on the JS layer. You could also look to initialize the native SDKs yourself to be able to provide custom options above what the React Native SDK currently provides.
+
+To do so, you would set [autoInitializeNativeSdk](/platforms/react-native/configuration/options/#autoInitializeNativeSdk) to `false` in the init options:
+
+```javascript
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  autoInitializeNativeSdk: false,
+});
+```
+
+This will prevent the React Native SDK from initializing the native SDK automatically.
+
+<Alert level="info" title="Note">
+
+Do not use `autoInitializeNativeSdk` to disable the native layer, instead use [enableNative](/platforms/react-native/configuration/options/#enableNative)
+
+</Alert>
+
+You will then have to initialize the native SDKs yourself following the guides for Android or iOS. You do not need to install any of the native SDKs by yourself as they are already packaged with the React Native SDK.
+
+## Android Configuration
+
+To use auto-init, you would have to add the following to your `AndroidManifest.xml`.
+
+```xml
+<meta-data android:name="io.sentry.auto-init" tools:replace="android:value" android:value="true" />
+```
+
+then you can follow [the guide on initializing Android](/platforms/android/#configure).
+
+<Note>
+
+Initializing via the above method uses a Content Provider; because the Javascript engine for React Native is also initialized via a Content Provider we cannot guarantee that Sentry would always initialize before the Javascript engine.
```

Comment:
we avoid latin terms

```suggestion
Initializing with the above method uses a Content Provider. Because the Javascript engine for React Native is also initialized using a Content Provider, we cannot guarantee that Sentry will always initialize before the JavaScript engine.
```